setup.sh

```diff
@@ -3,14 +3,53 @@ GREEN='\033[0;32m'
 CYAN='\033[0;36m'
 NC='\033[0m' # No Color
 
-printf "${CYAN}Get UI Bakery license - ${GET_KEY_LINK}:\n${NC}"
-printf '\n'
-printf "Enter license key:\n"
-while read license; do
-  test "$license" != "" && break
-  printf "${RED}License key is required!${NC}\n"
-  printf "Enter license key:"
+printf "${CYAN}Starting license setup...\n${NC}"
+echo "Do you already have a UI Bakery license key?"
+echo "Y - enter your license key, N - start a free trial."
```

Comment:
```suggestion
echo "Y - enter your license key, N - start a free trial. (Y)"
```
or something like that to indicated that Yes is a default.

Reply:
fixed